Assets/Scripts/Services/Input/InputController.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using Zenject;
+
+namespace Services.Input
+{
+    public class InputController : IInputController
+    {
+        private readonly InputActionAsset _inputActionAsset;
+        
+        public event Action<Vector2> MoveChanged;
+        public event Action<Vector2> FireChanged;
+        public event Action<Vector2> SpecialChanged;
+        public event Action MeleeChanged;
+        public event Action UseChanged;
+        public event Action ReloadChanged;
+
+        [Inject]
+        public InputController(InputActionAsset inputActionAsset)
+        {
+            _inputActionAsset = inputActionAsset;
+            OnEnable();
+        }
+
+        private void Move(InputAction.CallbackContext ctx)
+        {
+            MoveChanged?.Invoke(Vector2.zero);
+        }
+
+        private void Fire(InputAction.CallbackContext ctx)
+        {
+            FireChanged?.Invoke(Vector2.zero);
+        }
+
+        private void Special(InputAction.CallbackContext ctx)
+        {
+            SpecialChanged?.Invoke(Vector2.zero);
+        }
+
+        private void Melee(InputAction.CallbackContext ctx)
+        {
+            MeleeChanged?.Invoke();
+        }
+
+        private void Use(InputAction.CallbackContext ctx)
+        {
+            UseChanged?.Invoke();
+        }
+
+        private void Reload(InputAction.CallbackContext ctx)
+        {
+            ReloadChanged?.Invoke();
+        }
+
+        private void OnEnable()
+        {
+            _inputActionAsset.FindAction("Move").performed += Move;
```

Comment:
Pls extract string value to const.By our convention, your implementation falls under the magic number/value rule.
Also select the rest of the strings as constants

Reply:
Done